storage/memory/registers.go

```diff
@@ -0,0 +1,93 @@
+package memory
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/pkg/errors"
+	"github.com/rs/zerolog"
+	"go.uber.org/atomic"
+	"golang.org/x/exp/maps"
+	"golang.org/x/exp/slices"
+
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/storage"
+)
+
+var _ storage.RegisterIndex = (*Registers)(nil)
+
+type Registers struct {
```

Comment:
what is this for?

Reply:
I needed this until we have pebble for integration tests and manual tests. But it's valid point it can be removed from this PR.